gazebo_plugins/test/worlds/gazebo_ros_wheel_slip.world

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
+  <world name="default">
+    <gravity>-2 0 -9.8</gravity>
+
+    <include>
+      <uri>model://ground_plane</uri>
+    </include>
+    <include>
+      <uri>model://sun</uri>
+    </include>
+
+    <include>
+      <uri>model://trisphere_cycle</uri>
+      <name>trisphere_cycle_slip</name>
+      <pose>0 2 0  0 0 0</pose>
+      <plugin name="wheel_slip" filename="libgazebo_ros_wheel_slip.so">
+        <ros>
+          <namespace>trisphere_cycle_slip</namespace>
+        </ros>
+        <wheel link_name="wheel_front">
+          <slip_compliance_lateral>1</slip_compliance_lateral>
+          <slip_compliance_longitudinal>1</slip_compliance_longitudinal>
+          <wheel_normal_force>32</wheel_normal_force>
+        </wheel>
+        <wheel link_name="wheel_rear_left">
+          <slip_compliance_lateral>1</slip_compliance_lateral>
+          <slip_compliance_longitudinal>1</slip_compliance_longitudinal>
+          <wheel_normal_force>32</wheel_normal_force>
+        </wheel>
+        <wheel link_name="wheel_rear_right">
+          <slip_compliance_lateral>1</slip_compliance_lateral>
+          <slip_compliance_longitudinal>1</slip_compliance_longitudinal>
+          <wheel_normal_force>32</wheel_normal_force>
+        </wheel>
+      </plugin>
+      <plugin name="joint_control" filename="libJointControlPlugin.so">
+        <controller type="position">
+          <joint>wheel_front_steer</joint>
+          <target>0</target>
+          <pid_gains>9 0 0.1</pid_gains>
+        </controller>
+        <controller type="velocity">
+          <joint>wheel_front</joint>
```

Comment:
I believe the proper joint name here is `wheel_front_spin`

https://github.com/osrf/gazebo_models/blob/bd7bc758fdf82be65b545d614d6a85359ff0e47c/trisphere_cycle/model.sdf#L279

Reply:
I'm not sure why but it seems to be `wheel_front` for me in my local testing. Otherwise, I don't see the front wheel in the publisher. `wheel_front` matches the demo world here: https://github.com/ros-simulation/gazebo_ros_pkgs/blob/f642be7a206cfecddf4aaf09c033562a201c6592/gazebo_plugins/worlds/gazebo_ros_wheel_slip_demo.world#L35.

Reply:
I see, good catch.

Reply:
Fixed in https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1331/commits/7608cef133764af35d83862c5b6cbf186ce31667.